.github/workflows/buildsCI.yaml

```diff
@@ -258,8 +258,8 @@ jobs:
   
 
   #--------------------------------------------------------------------------------
-  U18_SH_RSUSB_LiveTest:  # Ubuntu 2018, Shared, Legacy live-tests
-    runs-on: ubuntu-18.04     
+  U20_SH_RSUSB_LiveTest:  # Ubuntu 2020, Shared, Legacy live-tests
```

Comment:
Log file names should be updated. See [example](https://github.com/IntelRealSense/librealsense/pull/11627/files#diff-c0d67cc2a479bca5448d7ad8cbe5c98cf7c7300f15225b13d859083d4caee221R312)